src/workers/sw.ts

```diff
@@ -108,9 +108,9 @@ registerRoute(/\/stored\/.*/, async (options) => {
 });
 
 registerRoute(
-  /^https:\/\/cdn.jsdelivr.net\/npm/,
+  /^https?:\/\/(cdn\.jsdelivr\.net|texlive2\.swiftlatex\.com)/,
```

Comment:
I'm fine with caching them but just make sure you know that in the deployed version, LaTeX dependencies are not served by this cache.

Reply:
Hmm why so?

Reply:
Here I set the path to be our own website: https://github.com/UCLA-IRL/ndn-workspace-solid/blob/38f1b0f0184871240e047f41d0f852995076a14a/src/components/share-latex/share-latex/index.tsx#L179-L182
I think we can have more control if we use OPFS. For example, when we want to support something like loading all packages in a zip file, we will be able to do so. Also, I don't think those files will be updated frequently.